deployments/data-hub/release-data-hub--prod.yaml

```diff
@@ -212,6 +224,18 @@ spec:
       - name: twitter-api-secret-volume
         mountPath: /dag_secret_files/twitter_api/
         readOnly: true
+      - name: twitter-api-key-secret-volume
+        mountPath: /dag_secret_files/twitter_api_key/
```

Comment:
I suppose since we are relying on the keys within the secret already, we could also store all four values in the same secret as separate keys / filenames?